source/_docs/style-guide.md

```diff
@@ -469,6 +469,35 @@ Use panels for extraneous but useful information such as troubleshooting section
 ```
 </div>
 
+## Tables
+
+Markdown tables are defined like this:
+
+<div class="style-example" markdown="1">
```

Comment:
I revised this table to refer to the corresponding HTML entity (e.g., table header, table data) but once we have the context improved in the intro.. it would be best to show an example of actual usage. Like what we do in the [error messages section](https://pantheon.io/docs/style-guide/#error-messages).